packages/metrics/chain.go

```diff
@@ -713,3 +716,7 @@ func (m *ChainMetricsProvider) OutPullTxInclusionState() IMessageMetric[iotago.T
 func (m *ChainMetricsProvider) OutPullOutputByID() IMessageMetric[iotago.OutputID] {
 	return m.outPullOutputByIDMetrics
 }
+
+func (m *ChainMetricsProvider) MaxChainConfirmedStateLag() uint32 { // TODO: Call it from the health endpoint.
```

Comment:
```suggestion
func (m *ChainMetricsProvider) MaxChainConfirmedStateLag() uint32 {
```
TODO can be removed?